### PR TITLE
*: automate building operator

### PR DESCRIPTION
**Pull request by LiliC**
Add make build and update .gitignore file.

Makefile will also be useful later on in the future as we will likely extend this to build binaries and container images. For now it just makes life a bit easier. :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,4 @@
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
 
-cmd/operator/operator
+operator
```
Comment (krnowak) [inline .gitignore]: I'd prefer to have `/operator`, so we don't accidentally ignore all the `operator` files/directories in the directory tree.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+build:
+	go build -i github.com/kinvolk/habitat-operator/cmd/operator
```
Comment (krnowak) [inline Makefile]: Add `.PHONY: build` below, so if you create a `build` file, then make will know to ignore it and run the build rule.

Comment (krnowak) [inline Makefile]: What this will do anyway? Will it download the project and build it instead of building the sources from this directory?

Comment (LiliC) [inline Makefile]: No, it expects it to be in your GOPATH and it builds from there.

**Comment by LiliC** [comment]
@krnowak Can you have another look, thanks.

**Comment by krnowak** [comment]
LFAD.

**Pull request merged**